parm/config/config.defaults.s2sw

```diff
@@ -16,13 +16,13 @@ use_cice_alb=".true."
 
 # config.wave
 
-waveGRD='mx025'
-waveinterpGRD='reg025'
-waveuoutpGRD='mx025'
-MESH_WAV='mesh.mx025.nc'
+waveGRD='gefsv13_025'
+waveinterpGRD=' '
+waveuoutpGRD='gefsv13_025'
```

Comment:
We will seek consistency when declaring names that belong to a category.
Some places we see `mx025`, some places just `025` for e.g.

Reply:
Then all the wave model grids need to be renamed.    If you tell me your preferred naming convention, then I'll update all the wave grids to match.  This will be a major fix file update FYI.  For grids already in operations, perhaps we can make exceptions though?

Reply:
`develop` is not going into **current** operations, so I don't think `gfsv16.x` needs to worry about this.

We don't know yet if **all** wave model grids will need updating either.
If we know what are the current wave model grids are, we can come to a consensus on the naming convention, retaining whatever consistency we have.
Please create an issue and this can be discussed there.

A list of the current `fix/wave` are:
```
.../fix/wave/
├── mesh.gefsv13_025.nc
├── mesh.gwes_30m.nc
├── rmp_src_to_dst_conserv_002_001.nc
├── rmp_src_to_dst_conserv_003_001.nc
├── weights_rtofs_to_r4320x2160.nc
├── WHTGRIDINT.bin.ak_10m
├── WHTGRIDINT.bin.glo_15mext
├── WHTGRIDINT.bin.glo_15mxt
├── WHTGRIDINT.bin.glo_30m
├── WHTGRIDINT.bin.reg025
├── ww3_grid.inp.ak_10m
├── ww3_grid.inp.ant_9km
├── ww3_grid.inp.ao_20m
├── ww3_grid.inp.aoc_9km
├── ww3_grid.inp.at_10m
├── ww3_grid.inp.ep_10m
├── ww3_grid.inp.gefsv13_025
├── ww3_grid.inp.glix_10m
├── ww3_grid.inp.glo_10m
├── ww3_grid.inp.glo_15m
├── ww3_grid.inp.glo_15mxt
├── ww3_grid.inp.glo_30m
├── ww3_grid.inp.glox_10m
├── ww3_grid.inp.gnh_10m
├── ww3_grid.inp.gsh_15m
├── ww3_grid.inp.gwes_30m
├── ww3_grid.inp.icean_5m
├── ww3_grid.inp.mx025
├── ww3_grid.inp.mx025gefs
├── ww3_grid.inp.mx050
├── ww3_grid.inp.mx100
├── ww3_grid.inp.points
├── ww3_grid.inp.reg025
├── ww3_grid.inp.reg050
├── ww3_grid.inp.reg100
├── ww3_grid.inp.rtofs_5m
├── ww3_grid.inp.so_20m
└── ww3_grid.inp.wc_10m
```

Most of them seem to have a consistent naming convention, with the exception of a couple.  We can catalog what these are, whether they are used or not, and provide backward compatibility for a year or two.